design/index.html

```diff
@@ -0,0 +1,112 @@
+---
+layout: main
+title: Design
+---
+
+{% capture title %}
+Learn design best practices
+{% endcapture %}
+
+{% capture text %}
+Use these resources to address user needs by focusing on user centred design, research and techniques.
+{% endcapture %}
+
+{% include digital_element_header.html title=title text=text %}
+
+{% capture content %}
+
+<div class="container col-md-10 text-center">
+  <h2>Develop your knowledge</h2>
+  <p class="lead">
+    Resources for improving your research and design skills, lovingly crafted by the team at Made Tech.
+  </p>
+</div>
+
+<div class="row">
+  <div class="col-sm">
+    <div class="card-body card-border">
+      <h3 class="card-title mt-0">User research</h3>
+      <p class="card-text">
+        Learn various techniques on how to conduct iterative user research effectively.
+      </p>
+      <a href="/" class="btn btn-success">Learn more</a>
```

Comment:
You can deaden links with `href="#"`

Reply:
Resolved in https://github.com/madetech/learn/pull/160/commits/e4f752906ba74a8140cf205bc07e24f3150633e5.